src/main/java/com/hobbycollection/HobbyCollectionController.java

```diff
@@ -13,4 +13,13 @@
     public String index() {
         return "start";
     }
+
+    /**
+     * Handle the AddEditCollection endpoint
+     * @return AddEditCollection
+     */
+    @RequestMapping("/addEditCollection")
+    public String addEditCollection() {
+        return "AddEditCollection";
+    }
```

Comment:
Added an endpoint that can be used to open the AddEditCollection.html page.

Citation: Lectures

Reply:
Is there an HTML page with this name?  If not, this is incomplete fuctionality.